apps/boundary/views.py

```diff
@@ -53,3 +56,33 @@ def get_queryset(self):
         else:
             queryset = queryset.filter(Q(boundary_type=BoundaryType.SCHOOL_CLUSTER)| Q(boundary_type=BoundaryType.PRESCHOOL_CIRCLE))
         return queryset
+
+class Admin2sInsideAdmin1(views.KLPListAPIView):
+    ''' Returns a list of all blocks/projects inside a given district id '''
+    serializer_class=BoundarySerializer
+
+    def get_queryset(self):
+        parent_district_id=self.kwargs.get('id',0)
+        print("Parent district id is: ", parent_district_id)
+        result= Boundary.objects.all().filter(
+            parent=parent_district_id,status='AC'
+        )
+        print("REsult is: ", result)
+        return result
+
+class Admin3sInsideAdmin1(views.KLPListAPIView):
+    '''Returns a list of all clusters/circles inside a given pre or primary district '''
+    serializer_class=BoundarySerializer
+
+    def get_queryset(self):
+        parent_district_id=self.kwargs.get('id',0)
+        return Boundary.objects.all().filter(parent__parent=parent_district_id,status='AC')
+
+class Admin3sInsideAdmin2(views.KLPListAPIView):
+    ''' Returns a list of all clusters/circles inside a given block/project id '''
+    serializer_class=BoundarySerializer
+    def get_queryset(self):
+        admin2_id=self.kwargs.get('id',0)
+        return Boundary.objects.all().filter(
```

Comment:
You can just do Boundary.objects.filter(). No need for Boundary.objects.all().filter()